en/FlyView/FlyView.md

```diff
@@ -1,79 +1,310 @@
+
+> **Note** You cannot pause a *Goto location* operation.
+
+To pause:
+1. Click the **Pause** button in the *Fly Tools*.
+1. Optionally set a new altitude using the right-side vertical slider.
+1. Confirm Pause using the slider.
+
+![pause](../../assets/fly/pause.jpg)
+
+
+### Missions
+
+#### Start Mission {#start_mission}
+
+You can start a mission when the vehicle is landed (the start mission confirmation slider is often displayed by default).
+
+To start a mission from landed:
+1. Click the **Action** button on the *Fly Tools*
+1. Select the *Start Mission* action from the dialog.
+
+   ![Start mission action](../../assets/fly/start_mission_action.jpg)
+   
+    (to display the confirmation slider)
+1. When the confirmation slider appears, drag it to start the mission.
+
+   ![Start mission](../../assets/fly/start_mission.jpg)
+
+#### Continue Mission {#continue_mission}
+
+You can *continue* mission from the *next* waypoint when you're flying (the *Continue Mission* confirmation slider is often displayed by default after you takeoff).
+
+> **Note** Continue and [Resume mission](#resume_mission) are different!
+  Continue is used to restart a mission that has been paused, or where you have taken off, so you've already missed a takeoff mission command.
+  Resume mission is used when you've used a RTL or landed midway through a mission (e.g. for a battery change) and then wish to continue the next mission item (i.e. it takes you to where you were up to in the mission, rather than continuing from you place in the mission).
+
+You can continue the current mission while (unless already in a mission!):
+1. Click the **Action** button on the *Fly Tools*
+1. Select the *Continue Mission* action from the dialog.
+
+   ![Continue Mission/Change Altitude action](../../assets/fly/continue_mission_change_altitude_action.jpg)
+
+1. Drag the confirmation slider to continue the mission.
+
+   ![Continue Mission](../../assets/fly/continue_mission.jpg)
+
+
+#### Resume Mission {#resume_mission}
+
+*Resume Mission* is used to resume a mission after performing an [RTL/Return](#rtl) or [Land](#land) from within a mission (in order, for example, to perform a battery change).
+
+> **Note** If you are performing a battery change, **do not** disconnect QGC from the vehicle after disconnecting the battery.
+After you insert the new battery *QGroundControl* will detect the vehicle again and automatically restore the connection. 
+
+After reconnecting you will be prompted with a *Resume Mission* confirmation slider.
+If you want to resume the mission, confirm this and the mission will start from the last waypoint traveled through. 
+Once the mission is rebuilt you will be presented with another Resume Mission slide which allows you to review the rebuilt mission before starting it again.
+Confirm this *Resume Mission* slider to continue on with the mission.
+
+![Resume Mission](../../assets/fly/resume_mission.jpg)
```

Comment:
Note, the image here is from a Land. Couldn't get RTL to pop up option - it just gave me option to remove the plan. This is on SITL and jmavsim.

Reply:
> Couldn't get RTL to pop up option

Not quite sure what you mean? Are you talking about not getting a resume mission to show up? You need to RTL from the middle of a mission. You have to be at least a couple waypoints away from the end when you do it.

Reply:
Yes. I thought I was - but in any case it all works now. Thanks.